src/includes/custom-metrics-supported-sdks.mdx

```diff
@@ -0,0 +1,7 @@
+- [JavaScript (version `7.0.0` or later)](/platforms/javascript/performance/instrumentation/performance-metrics/)
+- [Python (version `1.5.12` or later)](/platforms/python/performance/instrumentation/performance-metrics/)
+- [Apple (version `7.28.0` or later)](/platforms/apple/performance/instrumentation/performance-metrics/)
+- [Android (version `6.5.0` or later)](/platforms/android/performance/instrumentation/performance-metrics/)
+- [Dart and Flutter (version `6.11.0` or later)](/platforms/dart/performance/instrumentation/performance-metrics/)
```

Comment:
Let's split this into 2 items (similar to Java and Android), since Flutter has its own auto-perf metrics.
See PR https://github.com/getsentry/sentry-docs/pull/5695